virtwho/manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -179,7 +179,8 @@ def hypervisorCheckIn(self, report, options=None):
 
         is_async = self._is_rhsm_server_async(report, connection)
         serialized_mapping = self._hypervisor_mapping(report, is_async, connection)
-        self.logger.debug("Host-to-guest mapping: %s", json.dumps(serialized_mapping, indent=4))
+        self.logger.debug("Host-to-guest mapping being sent to '%s': %s", report.config.owner,
```

Comment:
@vritant I believe we should still (at least on the debug level) be dumping the hypervisor-guest mapping being sent. In the location where we are doing the counting (and logging at the INFO level), I can make the change requested (and use str.format if so desired).

Because the reports are batched (and we have no name for the batched reports sent) we can't change just that line.